storage/gc_queue.go

```diff
@@ -54,8 +60,10 @@ const (
 //
 //  - GC of version data via TTL expiration (and more complex schemes
 //    as implemented going forward).
-//  - Resolve extant write intents and determine oldest non-resolvable
-//    intent.
+//  - Resolve extant write intents (pushing their transactions).
+//  - GC of old transaction and sequence cache entries. This should include
+//    most committed entries almost immediately and, after a threshold on
+//    inactivity, all others.
 //
 // The shouldQueue function combines the need for both tasks into a
```

Comment:
s/both/all three/

`shouldQueue` needs to be updated to account for the number of GC'able transaction records.